storage-access.bs

```diff
@@ -147,6 +147,12 @@ partial interface Document {
 };
 </pre>
 
+Let |get unpartitioned cookie enabled state| be an algorithm that, given {{Document}} |doc| and [=cookie store=], runs the following steps:
```

Comment:
```suggestion
To <dfn>get unpartitioned cookies enabled state</dfn>, given a {{Document}} |doc|, run the following steps:
```

Others might feel differently, but I don't think it's worth passing around the cookie store here, given that it's an IETF concept that is only loosely translated to browser specs as this kind of global thing each browser maintains. Cookie layering will help here.